images/fabric-tools-ca/Dockerfile

```diff
@@ -1,19 +1,20 @@
-FROM hyperledger/fabric-tools:1.4.6
+FROM hyperledger/fabric-tools:1.4.9
```

Comment:
Why don't you start from https://hub.docker.com/r/hyperledger/fabric-ca-tools here ? Also if you could rename fabric-tools-ca to fabric-ca-tools that would be awsome :P (sorry for not catching it on the first review)

Reply:
Because it wasn't updated since two years.
Yes I can do the rename :)

Reply:
Good point! :P